pkg/proxy/iptables/proxier.go

```diff
@@ -1264,27 +1271,31 @@ func (proxier *Proxier) syncProxyRules() {
 							allowFromNode = true
 						}
 					}
-					// generally, ip route rule was added to intercept request to loadbalancer vip from the
-					// loadbalancer's backend hosts. In this case, request will not hit the loadbalancer but loop back directly.
-					// Need to add the following rule to allow request on host.
+					// Generally, an IP route rule was added to intercept
+					// requests to the LB VIP from the loadbalancer's backend
+					// hosts. In this case, request will not hit the
+					// loadbalancer but loop back directly.  We need the
+					// following rule to allow requests from this host.
```

Comment:
if you're going to re-wrap and re-capitalize this you should rewrite it to actually make sense too. :slightly_smiling_face: